client/src/styles/task.css

```diff
@@ -6,13 +6,25 @@
     gap: 4rem;
 }
 
+.task {
+        overflow-y: scroll;
```

Comment:
indentation is off here, perhaps consider using [Prettier](https://prettier.io/) as a code formatter